thrift/src/test/java/com/linecorp/armeria/it/metric/DropwizardMetricsIntegrationTest.java

```diff
@@ -72,16 +67,13 @@ protected void configure(ServerBuilder sb) throws Exception {
     private static final ClientFactory clientFactory =
             ClientFactory.builder().meterRegistry(registry).build();
 
-    @AfterClass
-    public static void closeClientFactory() {
+    @AfterAll
+    static void closeClientFactory() {
         clientFactory.close();
     }
 
-    @Rule
-    public final TestRule globalTimeout = new DisableOnDebug(new Timeout(30, TimeUnit.SECONDS));
```

Comment:
This is just gone?

Reply:
Now, we are using JUnit5 global timeout configar. https://github.com/line/armeria/pull/2425/files#diff-9c2d5d92bbbb9e98c96cf3f21ecedf97

Reply:
oh, that is a nice feature!